webpack/containers/Application/overrides.scss

```
@@ -75,3 +75,38 @@ html .pagination-pf-pagesize.btn-group {
     margin-left: 200px;
   }
 }
+
+@keyframes hideme {
+  0% {
+    opacity: 1;
+  }
+
+  10% {
+    opacity: 1;
+  }
+
+  30% {
+    opacity: 1;
+  }
+
+  80% {
+    opacity: 1;
+  }
+
+  90% {
+    opacity: 1;
+  }
+
+  100% {
+    opacity: 0;
+  }
```

I [learned](https://css-tricks.com/snippets/css/keyframe-animation-syntax/) that you can comma-separate values so it's a bit less repetitive.  Also I think it looks nicer if the fadeout starts at 70% instead of 90:
```suggestion
  0%, 70% {
    opacity: 1;
  }

  100% {
    opacity: 0;
  }
```